packages/react-native/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/react-native",
-  "version": "0.0.1",
+  "version": "0.1.0-alpha.0",
```

Comment:
is it ok to start react-native from 0.1.0 or from 0.0.1 ?

Reply:
good spot, thanks